app/services/settings_service/repository_base.rb

```diff
@@ -30,7 +30,11 @@ def create_table(name:)
     end
 
     def dynamodb
-      @dynamodb || Aws::DynamoDB::Client.new
+      if ENV['RAILS_ENV'] == 'development'
```

Comment:
I'm wondering if there is a cleaner way to write this? Something like
```
@dynamodb ||= Rails.env.development? ? Aws::DynamoDB::Client.new(:endpoint => "http://dynamodb:8000/") : Aws::DynamoDB::Client.new
```

I don't know if that will actually work or read better in practice so I could be swayed either way. Just a thought so we don't have multiple `@dynamodb ||=` statements...

Reply:
Curious if @adiaz04 has an opinion...

Reply:
I like it. 👍